frontend/ui/widget/opdsbrowser.lua

```diff
@@ -527,7 +527,7 @@ end
 
 function OPDSBrowser:createNewDownloadDialog(path, buttons)
     self.download_dialog = ButtonDialogTitle:new{
-        title = _("Choose format to download into:") .. "\n" .. path,
+        title = _("Download directory:") .. "\n" .. path .. "\n\n" .. "Download file type:",
```

Comment:
```lua
util.template(_("Download directory:\n%1\n\nDownload file type:"), path)
```

Or probably better to add `local T = require("ffi/util").template` to the top and go for the standard
```lua
T(_("Download directory:\n%1\n\nDownload file type:"), path)
```